client-core/src/wallet/default_wallet_client.rs

```diff
@@ -620,3 +636,31 @@ where
         self.transaction_builder.obfuscate(signed_transaction)
     }
 }
+
+fn parse_feedback(feedback: Option<&Feedback>) -> String {
+    match feedback {
+        None => "No feedback available!".to_string(),
+        Some(feedback) => {
+            let mut feedback_str = String::new();
+
+            if let Some(warning) = feedback.warning() {
+                feedback_str.push_str(&format!("Warning: {} | ", warning));
+            }
+
+            for suggestion in feedback.suggestions() {
+                feedback_str.push_str(&format!("Suggestion: {} | ", suggestion));
+            }
+
+            if feedback_str.is_empty() {
+                feedback_str = "No feedback available!".to_string();
+            } else {
+                // Pop last three chars
```

Comment:
I guess we can (join)[https://doc.rust-lang.org/1.30.0/std/slice/trait.SliceConcatExt.html#tymethod.join] here.

Reply:
Done